mysqlnd_azure.c

```diff
@@ -471,6 +533,9 @@ MYSQLND_METHOD(mysqlnd_azure_data, connect)(MYSQLND_CONN_DATA ** pconn,
 
 after_conn:
     {
+      if (MYSQLND_AZURE_G(logLevel) > 2) {
+        AZURE_LOG("DEBUG", "Prepare the context of new connection");
```

Comment:
after_conn will set host/user/xx field to the conn object and run init_commands and then return to up level. Maybe "Post operations after connection establishment" here?

Reply:
> Post operations after connection establishment

What do you mean?

Reply:
I mean here the handshake already finished, it will only do some post operation like fill the field for the conn object, run init_command. Prepare context looks for me means something before handshake. You may keep it if you think it is just fine.